packages/vuetify/src/components/VRangeSlider/VRangeSlider.ts

```diff
@@ -83,7 +84,7 @@ export default VSlider.extend({
   },
 
   methods: {
-    getTrackStyle (startLength: number, endLength: number, startPadding = 0, endPadding = 0) {
+    getTrackStyle (startLength: number, endLength: number, startPadding = 0, endPadding = 0): Partial<CSSStyleDeclaration> {
```

Comment:
No need for `: Partial<CSSStyleDeclaration>`

Reply:
Copied from VSlider.ts. Should I leave it there?

Reply:
no

Reply:
fixed